packages/vuetify/src/components/VExpansionPanel/VExpansionPanel.sass

```diff
@@ -21,6 +21,11 @@
     .v-expansion-panel-header
       +states($material)
 
+  &.v-expansion-panels--hover
+    .v-expansion-panel-header
```

Comment:
These styles will cascade to the inner panel, see https://github.com/vuetifyjs/vuetify/issues/9692 / https://github.com/vuetifyjs/vuetify/pull/9696

Reply:
yup, pushed an update